containers/buildenv.vscode/cmake-kits.json

```diff
@@ -0,0 +1,15 @@
+[
```

Comment:
Again, I don't think it's reasonable to start adding a bunch of files to special treat editors. You probably don't want me to dump my entire Emacs config in ;)

Reply:
Maybe not in the spectre repo, though I think a case could be made for support/DevEnvExamples or something like that, but some repo makes sense! You've emailed your enacts file on request, why not keep it around as an example for others?

Reply:
I do keep my emacs config around for others to use if interested: https://github.com/nilsdeppe/MyEnvironment (though the main reason is selfish: I can't lose the configs if they're in a repo ;) ).

I'm not against sharing the environment, I think `support/DevEnvExamples` is a reasonable place to put configs. I know SpEC has its Emacs input file syntax highlighting script somewhere, and I think that's a good thing. I'm not sure if the `json` files need to be in a specific location. If they do I think it's reasonable to have in the instructions "Copy file X to Y using `cp ...`". We can add where the file needs to get copied to the `.gitignore` list. Does that sound okay?

Reply:
Sounds perfect to me! We'll just add a readme saying "copy .vscode to $SPECTRE_HOME"inside support/DevEnvExamples/vscode

Would be nice if separately you put your example env in support/DevEnvExamples/emacs or whatever you want to name it, so it's easier to find, if you're willing to